py/desitarget/lyazcat.py

```diff
@@ -0,0 +1,463 @@
+"""
+desitarget.lyazcat
+==============
+
+Post-redrock ML processing for LyA Quasar object identification.
+"""
+
+import os
+import numpy as np
+import sys
+from astropy.io import fits
+import fitsio
+import time
+
+from desitarget.geomask import match, match_to
+from desitarget.internal import sharedmem
+from desitarget import io
+from desispec.io import read_spectra
+
+from quasarnp.io import load_model
+from quasarnp.io import load_desi_coadd
+from quasarnp.utils import process_preds
+from squeze.model import Model
+from squeze.common_functions import save_json, load_json
```

Comment:
I don't think you actually use `save_json` anywhere in the code, so there shouldn't be any need to import it?

Reply:
Correct. Thanks for catching it. I removed the import.